setup.py

```diff
@@ -40,7 +40,7 @@ def get_install_requires():
     include_package_data=True,
     exclude_package_data={"": ["development", "development.*"], "development": ["*"]},
     packages=setuptools.find_namespace_packages(
-        exclude=["development", "development.*"]
+        exclude=["development", "development.*", "tests*"]
```

Comment:
Should this be `tests*` or `test*`?  I don't think we have a tests directory?

Reply:
You are right, the folder is test